src/org/traccar/database/DeviceManager.java

```diff
@@ -316,7 +316,16 @@ public void removeGroup(long groupId) throws SQLException {
         groupsById.remove(groupId);
     }
 
-    public String lookupAttribute(long deviceId, String attributeName) {
+    public String lookupServerAttribute(long deviceId, String attributeName) {
```

Comment:
.. double as well.

Also I think  we use attributes with reserve. We should use them more in some protocols for timezones or chose subformat per device. Hope this helpers "help" in adoption.

There will be a lot of new functions (8 at least), leave them in DeviceManager or move attribute lookup to new class?